docs/_templates/footer.html

```diff
@@ -0,0 +1,7 @@
+{% extends "!footer.html" %}
+{% block extrafooter %}
```

Comment:
[Apparently](https://github.com/snide/sphinx_rtd_theme/issues/349) it adds to the existing footer block with `extrafooter` being an RTD block as [normal themes just use `footer`](http://www.sphinx-doc.org/en/latest/templating.html?highlight=template#jinja-sphinx-templating-primer).

I fought about right justified text or keep the button floating to the side of text and settled on that looking cleaner.